### ISSUE TITLE
Y24-007 Updates to PBMC donor pooling - create plate

### ISSUE THREAD

**Issue by yoldas** [sanger/limber#1611]
**User story**
As a user of the scRNA Core pipeline, I would like updates to the PBMC Donor pooling to support GEM-X chip.

**Who are the primary contacts for this story**
Katy, Lesley, Abby

**Who is the nominated tester for UAT**
Lesley, Abby

**Acceptance criteria**
To be considered successful the solution must allow:
- [x] Replace the lookup table for the number of source samples according to the chip type decided. Currently, [this table](https://github.com/sanger/limber/blob/develop/config/cardinal_pooling.csv) (copy of Cardinal's) is used for 1-96 samples and a default number (16) for the number of pools is assumed for more samples.
- [ ] Add the constraint around the collection sites, as with Cardinal (if any) to donor pooling. At the time of writing this story, the requirement is not specified.
- [ ] Update pooling algorithm for an even distribution of samples across pools. For example, for 32 samples with same study and project, and unique donors, it generates 3 pools as expected but the sizes are 8, 8, and 16. It is because of the halving of the biggest pools for distribution done by the algorithm. They should be 11, 11, and 10 instead. Only the distribute_groups_across_pools method needs changing.
- [x] Consider removing the pooling config file (look up table; Interim scRNA Core pooling config). A pooling strategy based on a suggested number of cells in a pool and a maximum may be better. For example, suggested size could be 10 and maximum could be 20.

**Dependencies**
This story is blocked by the following dependencies:
- https://github.com/sanger/limber/issues/1390

**Additional context**
Only one chip type is supported in this step. The lookup table to configure the number of pools based on the number of samples is affected by the decision of which chip to use and additional chip-specific considerations. We need to change code and configuration to use 10X Chromium 8-well GEM-X chip instead of 10X Chromium 16-well chip.

**Comment by KatyTaylor** [comment]
Still on hold, awaiting discussion around pooling strategy in Thursday's working group.

**Comment by SujitDey2022** [comment]
@KatyTaylor @yoldas is this story still on hold?

**Comment by KatyTaylor** [comment]
Yes, I believe so. We're still waiting to hear from R&D about it.

**Comment by KatyTaylor** [comment]
Update from Abby last Friday (26th July) was that they have made some progress on this but still haven't finalised the strategy.

**Comment by KatyTaylor** [comment]
This is covered by all the pooling stories we've created. 

The potential 'collection site' requirement might come back when Cardinal comes on board.
